src/pages/schema-reference.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useLocation } from '@docusaurus/router';
+import SchemaReference from '../components/SchemaReference';
+
+function useQuery() {
+  const { search } = useLocation();
+  return React.useMemo(() => new URLSearchParams(search), [search]);
+}
+
+const AVAILABLE_SCHEMAS = [
+  'Reader.schema.json',
+  'Builder.schema.json',
+  'ManifestDefinition.schema.json',
+  'Settings.schema.json',
+];
```

Comment:
Hm, I'm not a huge fan of hardcoding this here. Can we render all children of the schemas directory by default but then optionally choose what to display in the sidebars.js code? Not a huge deal, just something that may be more future-proof.